content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,424 @@
+---
+date: "2021-11-10T00:00:00+00:00"
+title: 'Interchange project update'
+tags: ["infrastructure", "interoperability"]
+categories: ["general"]
+draft: false
+description: 'Interchange: Enabling the broader use of OpenFF force fields'
+slug: 'interchange-update-2021-11-10'
+weight: 10
+markup: markdown
+thumb: "foo.png"
+author: "Matt Thompson"
+---
+
+
+Thus far, the only simulation engine natively supported by [export from the OpenFF software
+stack](https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.typing.engines.smirnoff.forcefield.ForceField.html#openff.toolkit.typing.engines.smirnoff.forcefield.ForceField.create_openmm_system)
+is been OpenMM. OpenMM offers great performance on GPUs and excellent flexibility via its API, but
+many scientists wish to use tools that are only compatible with other engines. There are tools that
+can convert OpenMM objects to file formats understood by other engines, but these steps can have
+rough edges and add complexity to users' workflows. The object of the Interchange project is to
+provide a first-class support for these operations with the OpenFF software stack. This enabled by
+a Python package of the same name that is described below and in its
+[documentation](https://openff-interchange.readthedocs.io/en/latest/index.html#).
+
+This post aims to provide a high-level overview of the aims of the project, its current status,
+present capabilities, and some future directions.
+
+{{< note >}}
+Like much of of OpenFF infrastructure, Interchange is rapidly evolving and details within this blog post are likely to be out of date in the future.
+{{< /note >}}
+
+Code in this post should run if copy-pasted into a Jupyter notebook or Python interpreter. This is
+being written in November 2021 and code snippets intended to be used with Interchange v0.1.3. The API is
+not yet stable; the names, import paths, and structure of key components will likely be updated in
+the future in accordance with user feedback. If you'd like to follow along, see installation
+instructions at the bottom of the page; ``Interchange`` is distributed under the MIT license and all
+functionality is accessible using only free and open-source tools.
+
+
+## Contents of an Interchange object
+
+To emphasize a detail above, the key objective of the Interchange project at present is to make it
+easier to get OpenFF force fields in use in non-OpenMM workflows. The scope of this task is large,
+but it is finite; the
+[SMIRNOFF](https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/users/smirnoff.html)
```

Comment:
Should point to standards repo since that's the new authoritative home of the SMIRNOFF spec